### ISSUE TITLE
Fetch available models from Web IO repositories.

### ISSUE THREAD

**Issue by achillesrasquinha** [opencobra/cobrapy#1284]
### Checklist

- [ ] There are [no similar issues or pull requests](https://github.com/opencobra/cobrapy/issues) for this yet.

## Describe the solution you would like.

Would be nice to have a model list of each repository to the Web IO repositories.

```
bigg = BiGGModels()
models_avail = bigg.model_list() # id list of models available.
```

**Comment by cdiener** [comment]
Yeah, would be useful. Can you see a way to do that quickly? For instance, Biomodels is huge...
